dt-impact-tracer/src/main/java/edu/washington/cs/dt/impact/util/RuntimeGenerator.java

```diff
@@ -130,18 +139,84 @@ public static void main(String[] args) throws IOException {
                             System.err.println(e.getMessage());
                         }
                     }
-                    csvPrinter.printRecord(name, endTest, "", testInfo[3].trim().split("#")[1], "");
+                    createeElemntend(doc,test,methods,testInfo[3].trim().split("#")[1]);
+                    root.appendChild(test);
                 }
-                fileReader.close();
-            } catch (FileNotFoundException e) {
+
+                doc.appendChild(root);
+                DOMSource source=new DOMSource(doc);
+                String path="sootXMLOutput" + File.separator + paths[paths.length - 4] + "-runtime.xml";
+                File f=new File(path);
+                Result result=new StreamResult(f);
+                TransformerFactory transformerFactory=TransformerFactory.newInstance();
+                Transformer transformer=TransformerFactory.newInstance().newTransformer();
+                transformer.setOutputProperty(OutputKeys.OMIT_XML_DECLARATION,"no");
+                transformer.setOutputProperty(OutputKeys.INDENT,"yes");
+                transformer.transform(source,result);
+
+
+
+            } catch (ParserConfigurationException e) {
+                System.err.println(e.getMessage());
+            } catch (TransformerConfigurationException e) {
+                System.err.println(e.getMessage());
+            } catch (TransformerException e) {
                 System.err.println(e.getMessage());
             }
-            csvPrinter.flush();
-        } catch (IOException e){
-            System.err.println(e.getMessage());
-        } finally {
-            fileWriter.close();
-            bufferedWriter.close();
+    }
+
+    private static void createeElemnthead(Document doc,Element test,String test_val,String execution_time_val)
```

Comment:
```suggestion
    private static void createElemnthead(Document doc,Element test,String test_val,String execution_time_val)
```
Also can you fix it where else it has been called

Reply:
@Talank Noted.